src/sentry/api/serializers/models/organization.py

```diff
@@ -653,10 +657,30 @@ def serialize(  # type: ignore[explicit-override, override]
             team__organization=obj
         ).count()
 
-        sample_rate = quotas.backend.get_blended_sample_rate(organization_id=obj.id)
-        context["isDynamicallySampled"] = (
-            has_dynamic_sampling(obj) and sample_rate is not None and sample_rate < 1.0
-        )
+        is_dynamically_sampled = False
+        sample_rate = None
+        if has_dynamic_sampling(obj):
+            if (
+                obj.get_option("sentry:sampling_mode", SAMPLING_MODE_DEFAULT)
+                == DynamicSamplingMode.ORGANIZATION
+            ):
```

Comment:
nit: do we want to extract this as a util `is_organization_mode_sampling` (like we have for `is_project_mode_sampling`) and have 2 simpler if-conditions, and less indentation :)